src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,116 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    static final int DEFAULT_INITIAL_CAPACITY = 16;
+    static final float DEFAULT_FACTOR = 0.75f;
+    private int currentCapacity;
+    private Node<K, V>[] table;
+    private int threshold;
+    private int size;
+
+    {
+        this.table = new Node[DEFAULT_INITIAL_CAPACITY];
+        this.currentCapacity = table.length;
+        this.threshold = (int) (currentCapacity * DEFAULT_FACTOR);
+    }
 
     @Override
     public void put(K key, V value) {
-
+        Node<K, V> newNode = new Node<>(key, value);
```

Comment:
Using public access modifiers everywhere: The 'Node' class should be private because it is an implementation detail of 'MyHashMap' and is not intended to be used outside of this class.